agent/agent.go

```diff
@@ -337,15 +294,9 @@ func (a *Agent) RecordManifest() {
 	a.ManifestOps = result
 }
 
-// WriteOutput renders the manifest and results of the diagnostics run and writes the compressed archive.
+// WriteOutput renders the manifest and results of the diagnostics run into Agent.tmpdir
 func (a *Agent) WriteOutput() (err error) {
-	err = util.EnsureDirectory(a.Config.Destination)
-	if err != nil {
-		a.l.Error("Failed to ensure destination directory exists", "dir", a.Config.Destination, "error", err)
-		return err
-	}
-
-	a.l.Debug("Writing results and manifest, and creating tar.gz archive")
+	a.l.Debug("Writing results and manifest")
```

Comment:
Since this is a debug, maybe we should include the location in it - that may help users who are troubleshooting some issue or other, if they don't know where the temp directory is already.